src/blocks/main/auth-form/__button/auth-form__button.css

```diff
@@ -0,0 +1,7 @@
+.button.auth-form__button {
+  height: 64px;
+  margin-top: 20px;
+  margin-bottom: 16px;
+  width: 100%;
+  justify-content: center;
+}
```

Comment:
There was not auth form requirement in the task description, it seems like an unused piece of code.